Makefile

```diff
@@ -118,7 +118,7 @@ gunicornserve: .venv/build.timestamp
 
 .PHONY: dockerrun
 dockerrun:
-	docker-compose up -d;
+	DTM_BASE_PATH=${DTM_BASE_PATH} HTTP_PORT=${HTTP_PORT} docker-compose up -d;
```

Comment:
The `HTTP_PORT` env is missing in the `shutdown` target, this mean that if we change the default port, the shutdown will still use the default one and will probably fail